routers/repo/issue.go

```diff
@@ -434,14 +434,193 @@ func retrieveProjects(ctx *context.Context, repo *models.Repository) {
 	}
 }
 
+// ReviewerChoseItem items to bee shown
+type ReviewerChoseItem struct {
```

Comment:
Should not be exported and name should be probably renamed
```suggestion
type repoReviewerSelection struct {
```